langlib/lang.value/src/main/ballerina/src/lang.value/value.bal

```diff
@@ -200,3 +200,10 @@ public isolated function fromJsonStringWithType(string str, typedesc<anydata> t)
 # - otherwise, the merge fails
 # If the merge fails, then `j1` is unchanged.
 public isolated function mergeJson(json j1, json j2) returns json|error = external;
+
+# Converts a field in map<json> to a user-specified basic simple type.
```

Comment:
While this is the use-case for which we need this method, this isn't the only usage, right? Shall we introduce docs describing the function in a generic manner?

Reply:
Once the spec is updated, we have to use the spec documentation.

Reply:
One option is to go without a doc until the spec is ready.

Reply:
Created issue for this(https://github.com/ballerina-platform/ballerina-lang/issues/26355).